training/src/main/scala/com/airbnb/aerosolve/training/LinearRankerUtils.scala

```diff
@@ -168,4 +181,14 @@ object LinearRankerUtils {
       CompressedExample(getFeatures(ex), Array[(String, String)](), label)
     })
   }
+  
+  def getLabel(fv : FeatureVector, rankKey : String, threshold : Double) : Double = {
+    val rank = fv.floatFeatures.get(rankKey).asScala.head._2
+    val label = if (rank <= threshold) {
+      -1.0
+    } else {
+      1.0
+    }
+    return label
+  }
 }
```

Comment:
These seem to be something that can be used for all models, why don't we move this to TrainingUtils